src/txmempool.cpp

```diff
@@ -892,6 +892,12 @@ bool CCoinsViewMemPool::GetCoin(const COutPoint &outpoint, Coin &coin) const {
     // conflict with the underlying cache, and it cannot have pruned entries (as it contains full)
     // transactions. First checking the underlying cache risks returning a pruned entry instead.
     CTransactionRef ptx = mempool.get(outpoint.hash);
+    if (!ptx) {
+        // If a coin is missing from the mempool, check to see if it's part of
+        // a candidate package
```

Comment:
Possibly mention here the re-assignment of `ptx` if part of a candidate package, something like:
```cpp
// a candidate package; if yes, reassign it from the package.
```